src/client/components/app/Navigation/NavSidebar.tsx

```diff
@@ -6,7 +6,6 @@ import { SettingsSelectors } from '@store';
 import { NavigationLink } from '@components/app';
 import { Link, Icon, Logo, RedirectIcon } from '@components/common';
 
-const linkHoverFilter = 'brightness(90%)';
```

Comment:
this removes hover color change in the themes: `cyberpunk, classic, explorer`.

Reply:
Also sidebar default color seems wrong on classic:
![image](https://user-images.githubusercontent.com/78362532/167504261-41a00fec-31d4-47d8-a055-6f12eba54dbc.png)

Reply:
classic is as per figma. i did notice Vaults LH is not active when user is in Vaults / VaultName

<img width="800" alt="image" src="https://user-images.githubusercontent.com/80323528/167603395-8915ee55-03b0-489a-a48b-bd00a18eb82c.png">

Ideally Vaults LH nav should remain active when anywhere in that section - i'll see if i can fix that

Reply:
hover is now specific colour

<img width="459" alt="image" src="https://user-images.githubusercontent.com/80323528/167606425-92dff0d8-ae15-4804-bb77-708ef8f752ac.png">

Reply:
> Also sidebar default color seems wrong on classic: ![image](https://user-images.githubusercontent.com/78362532/167504261-41a00fec-31d4-47d8-a055-6f12eba54dbc.png)

good catch - need to get figma aligned with codebase colour naming patterns (ie by object name, not colour name)

<img width="688" alt="image" src="https://user-images.githubusercontent.com/80323528/167606754-6f090a3b-1da4-4f5e-ac81-ce256f3b0ccd.png">

Reply:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/80323528/167703057-a4638d3b-4c51-4c93-b0ec-599f2563cb1d.png">

<img width="243" alt="image" src="https://user-images.githubusercontent.com/80323528/167703083-ebe39265-1153-4cc7-b484-a2517f5c582b.png">

<img width="369" alt="image" src="https://user-images.githubusercontent.com/80323528/167703112-df44f318-352a-4452-9abd-f1690dbf3b1c.png">